integration/js/checks/ContentShareVideoCheck.js

```diff
@@ -0,0 +1,50 @@
+const {KiteTestError, Status} = require('kite-common');
+const AppTestStep = require('../utils/AppTestStep');
+
+class ContentShareVideoCheck extends AppTestStep {
+  constructor(kiteBaseTest, sessionInfo, testType, videoIndex) {
+    super(kiteBaseTest, sessionInfo);
+    this.expectedState = testType === 'ON' ? 'video' : 'blank';
```

Comment:
If switch statement below is doing the same job, then why do we need this statement?